Specs/TestSpec/spec.yml

```diff
@@ -358,6 +358,15 @@ components:
           woof: "#/components/schemas/Dog"
           dog: "#/components/schemas/Dog"
           cat: "#/components/schemas/Cat"
+    SingleAnimalWithObjectType:
```

Comment:
Isn’t this the same as the above SingleAnimal?

Reply:
except it has the additional `type: object`

I wanted to make sure there is a test for both cases: with the `type: object` and without (the SingleAnimal)

Reply:
Ah I see you added it now 👍